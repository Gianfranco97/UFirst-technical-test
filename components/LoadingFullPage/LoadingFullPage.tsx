import { Spin } from 'antd';
import MainLayout from 'components/MainLayout';
import styles from "./LoadingFullPage.module.scss"

export default function LoadingFullPage() {
  return (
    <MainLayout>
      <div className={styles.loadingContainer}>
        <Spin size="large" />
      </div>
    </MainLayout>
  )
}