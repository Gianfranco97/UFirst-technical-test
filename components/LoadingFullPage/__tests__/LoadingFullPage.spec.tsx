import { expect } from '@jest/globals';
import { render } from '@testing-library/react'
import LoadingFullPage from '../LoadingFullPage';

describe('LoadingFullPage', () => {
  it('renders correctly', () => {
    const { container } = render(
      <LoadingFullPage />
    )

    expect(container).toMatchSnapshot()
  })
});