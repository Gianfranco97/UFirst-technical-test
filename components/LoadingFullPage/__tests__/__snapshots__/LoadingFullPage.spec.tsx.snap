// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingFullPage renders correctly 1`] = `
<div>
  <section
    class="ant-layout layout"
  >
    <header
      class="ant-layout-header header"
    >
      <h1>
        Technical test - UFirst
      </h1>
    </header>
    <section
      class="ant-layout"
    >
      <main
        class="ant-layout-content main"
      >
        <div
          class="loadingContainer"
        >
          <div
            aria-busy="true"
            aria-live="polite"
            class="ant-spin ant-spin-lg ant-spin-spinning"
          >
            <span
              class="ant-spin-dot ant-spin-dot-spin"
            >
              <i
                class="ant-spin-dot-item"
              />
              <i
                class="ant-spin-dot-item"
              />
              <i
                class="ant-spin-dot-item"
              />
              <i
                class="ant-spin-dot-item"
              />
            </span>
          </div>
        </div>
      </main>
    </section>
    <footer
      class="ant-layout-footer footer"
    >
      Â© 2022 - Gianfranco Manganiello
    </footer>
  </section>
</div>
`;
